src/StubFilesExtension.php

```
@@ -192,8 +235,81 @@ private function generateStub(string $applicationType): string
+
+            $entries = [];
+
+            foreach ($value as $v) {
+                $entries[] = $this->inferTypeString($v);
+            }
+
+            return 'array{' . implode(', ', $entries) . '}';
+        }
+
+        return 'mixed';
+    }
```

PHPMD (2.15.0)

[warning] The method inferTypeString() has a Cyclomatic Complexity of 14. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)